src/tilegarden/src/config/map-config.mml

```diff
@@ -12,10 +12,10 @@ Layer:
   geometry: polygon
   srs: "+proj=lcc +lat_1=40.96666666666667 +lat_2=39.93333333333333 +lat_0=39.33333333333334 +lon_0=-77.75 +x_0=600000 +y_0=0 +ellps=GRS80 +datum=NAD83 +to_meter=0.3048006096012192 +no_defs"
   Datasource:
-    host: ${TILEGARDEN_HOST}
-    dbname: ${TILEGARDEN_DB}
-    user: ${TILEGARDEN_USER}
-    password: ${TILEGARDEN_PASSWORD}
+    host: ${PFB_DB_HOST}
```

Comment:
This is a far reaching change that impacts upstream changes when we can just map the environment variable in the docker compose config. Are we sure we want to do this now?

The env munging change seems separate, and ok.

Reply:
The config files where this is used are already project-specific.  I think adding generality to how configuration is assembled (especially not having to repeat the whole database config for every layer) is probably something that should happen in Tilegarden, but as it stands right now changing these variables doesn't amount to changing anything very general.  So I think the benefit of not having two copies of these variables to define and keep in sync in PFB is enough that it's worth switching.

Reply:
Makes sense.